components/nodemanager-service/pgdb/node-managers-process-node.go

```diff
@@ -86,9 +168,47 @@ func (db *DB) ProcessIncomingNode(node *manager.NodeMetadata) error {
 		}
 		return tx.tagNode(node.GetUuid(), tags)
 	})
+
+	return err
+}
+
+func (db *DB) getCurrentScanStatus(node *manager.NodeMetadata) (string, error) {
+	var status sql.NullString
+	err := db.QueryRow(sqlGetCurrentScanStatus, node.GetUuid(),
+		node.GetSourceId(), node.GetSourceRegion(), node.GetSourceAccountId()).Scan(&status)
+	switch {
+	case err == sql.ErrNoRows:
+		return "", nil
+	case err != nil:
+		return "", errors.Wrap(err, "unable to query for current status")
+	}
+	if !status.Valid {
+		return "", nil
+	}
+	statusString, err := strconv.Unquote(status.String)
 	if err != nil {
-		return err
+		return "", errors.Wrap(err, "unable to read status")
 	}
+	return statusString, nil
+}
 
-	return err
+func (db *DB) getCurrentRunStatus(node *manager.NodeMetadata) (string, error) {
+	var status sql.NullString
+	err := db.QueryRow(sqlGetCurrentRunStatus, node.GetUuid(),
+		node.GetSourceId(), node.GetSourceRegion(), node.GetSourceAccountId()).Scan(&status)
+	switch {
+	case err == sql.ErrNoRows:
+		return "", nil
+	case err != nil:
+		return "", errors.Wrap(err, "unable to query for current status")
+
+	}
+	if !status.Valid {
+		return "", nil
+	}
+	statusString, err := strconv.Unquote(status.String)
```

Comment:
is there something else i should be using here instead of Unquote?

Reply:
What does the data look like before and after Unquote?

Reply:
`\"\passed\"\` and then the correct "passed".  Unquote works well there; I just wasn't sure if there was something else we should use instead